instrumentation/oracle-ucp-11.2/javaagent/build.gradle.kts

```diff
@@ -13,10 +13,15 @@ muzzle {
 
 dependencies {
   library("com.oracle.database.jdbc:ucp:11.2.0.4")
+  library("com.oracle.database.jdbc:ojdbc8:12.2.0.1")
```

Comment:
I didn't see any new imports?

Reply:
It is used indirectly in `connectionPool.setConnectionFactoryClassName("oracle.jdbc.pool.OracleDataSource");` Also in latest dep test the jdbc driver jar is needed to avoid a compiler warning that gets treated as an error.

Reply:
why not ojdbc11? 

Oracle JDBC Driver compatible with JDK11, JDK17, JDK19, and JDK21

ojdbc8 is for JDK 8 and JDK 11.

Reply:
We run tests on jdk8 so the jdbc driver that works on jdk8 suites us just fine. We don't really care about the jdbc driver here, we only want to test that the connection pool implementation produces metrics.